imports/both/i18n/en/phm/home.json

```
@@ -174,24 +274,16 @@
           "title": "#Community Climate Action Platform",
           "url": "https://publichappinessmovement.com/c/climate-action/",
           "image": "/images/cover_climate_hack.png",
-          "description": "A platform which brings people and communities together around effective action we as individuals can take to fight climate catastrophe, and a token which incentivises the most important fight humanity faces today",
+          "description": "A platform which asks people and communities to come together around effective action that we as individuals can take to fight climate change, and rewards all positive action taken with a crypto token, incentivising the most important fight of our lifetime",
           "categories": [
-	    {
+	          {
               "title": "Project Brief",
               "url": "https://publichappinessmovement.com/t/topic/2753/"
-            },
-	    {
-              "title": "Hack 1: Mid-May Location coming soon",
-              "url": "https://publichappinessmovement.com/c/climate-action/climate-hack1"
-            },
-            {
-              "title": "Hack 2: 4th - 17th July at Fort Kuddelstaart in Amsterdam",
-              "url": "https://publichappinessmovement.com/c/climate-action/climate-hack2/"
             }
           ]
         },
 		    {
-          "color": "red",
+          "color": "Emerald",
```

I don't think there's a color called Emerald that the browser would understand. You can get an alternate color here: https://htmlcolorcodes.com/color-names/, or use the color code of emerald which is the belo:

```suggestion
          "color": "#50C878",
```